Project.toml

```diff
@@ -1,22 +1,25 @@
 name = "Supergrassi"
 uuid = "524c9cd3-385d-434e-bae6-611159fb61cc"
-authors = ["Tuomas Koskela <t.koskela@ucl.ac.uk> and contributors"]
+authors = ["Tuomas Koskela <t.koskela@ucl.ac.uk> and Arindam Saha <arindam.saha@ucl.ac.uk>"]
 version = "1.0.0-DEV"
 
 [deps]
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 XLSX = "fdbf4ff8-1666-58a4-91e7-1b58723a45e0"
 YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 
 [compat]
-Aqua = "0.8"
 CSV = "0.10.15"
 DataFrames = "1.7.0"
-Test = "1"
+Distributions = "0.25.117"
+Enzyme = "0.13.30"
+Random = "1.11.0"
 XLSX = "0.10.4"
 YAML = "0.4.12"
-julia = "1.10"
```

Comment:
This should be kept :slightly_smiling_face:

Reply:
Oops :blush: